packages/benchmark/package.json

```
@@ -0,0 +1,70 @@
+{
+  "name": "@endo/benchmark",
+  "version": "0.1.0",
+  "private": true,
+  "description": "Endo benchmarking ",
+  "keywords": [],
+  "author": "Endo contributors",
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/endojs/endo/tree/master/packages/skel#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/endojs/endo.git",
+    "directory": "packages/benchmark"
+  },
+  "bugs": {
+    "url": "https://github.com/endojs/endo/issues"
+  },
+  "type": "module",
+  "main": "./index.js",
+  "module": "./index.js",
+  "exports": {
+    ".": "./index.js",
+    "./package.json": "./package.json"
+  },
+  "scripts": {
+    "build": "exit 0",
+    "lint": "yarn lint:types && yarn lint:eslint",
+    "lint-check": "yarn lint",
+    "lint-fix": "yarn lint:eslint --fix && yarn lint:types",
+    "lint:eslint": "eslint '**/*.js'",
+    "lint:types": "tsc",
+    "postpack": "git clean -f '*.d.ts*'",
+    "prepack": "tsc --build tsconfig.build.json",
+    "test": "./run-tests.sh",
+    "bench": "yarn rollup -c && node --jitless --cpu-prof dist/bundle.js"
```

I find it very strange for `yarn bench` to run `node --cpu-prof` against the hard-coded test/index.test.js file.
```suggestion
    "cpu-prof-test": "yarn rollup -c && node --jitless --cpu-prof dist/bundle.js"
```